roles/configure_jenkins_projects/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
     dest: "{{ jenkins_xml_dest }}/{{ item }}_config.xml"
     owner: jenkins
     group: jenkins
-    mode: '0740'
+    mode: '0755'
```

Comment:
This does not make sense to me. An XML file should never be executable. Hence I would expect `0600` or `0660` or maybe `0664` if it must be read by others then the jenkins user...